### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# FROM fuzzers/go-fuzz:1.2.0
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Please rename from fuzz.libfuzzer to something more descriptive

File: mayhem/fuzz.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package fuzz
```
Comment (xansec) [inline mayhem/fuzz.go]: Please rename to indicate what's being fuzzed.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**